src/courses.json

```diff
@@ -0,0 +1,326 @@
+[
+  {
+    "faculty": "Górnictwa i Geoinżynierii",
```

Comment:
Ahh, right. It's .json file. Not object in .js file.
In this case you cannot do without double quotes. :)